.github/scripts/test_tryrebase.py

```diff
@@ -22,6 +22,21 @@ def test_rebase(self, mocked_post_comment: Any, mocked_run_git: Any, mocked_gql:
         mocked_run_git.assert_has_calls(calls)
         self.assertTrue("Successfully rebased `master` onto `master`" in mocked_post_comment.call_args[0][3])
 
+    @mock.patch('trymerge.gh_graphql', side_effect=mocked_gh_graphql)
+    @mock.patch('gitutils.GitRepo._run_git')
+    @mock.patch('tryrebase.gh_post_comment')
+    def test_rebase_to_stable(self, mocked_post_comment: Any, mocked_run_git: Any, mocked_gql: Any) -> None:
+        "Tests rebase to viable/strict successfully"
+        pr = GitHubPR("pytorch", "pytorch", 77907)
```

Comment:
Can we use an older PR so we don't need to add to gql_mocks.json lol

Reply:
aka one that already existed (like would 31093 work?)

Reply:
changed the PR to 31093, but it still says that gql_mocks.json was changed

Reply:
nvm fixed!